src/constants.js

```diff
@@ -350,6 +350,7 @@ export const TASK_REJECTED: 'rejected' = 'rejected';
 export const TASK_NEW_APPROVED: 'APPROVED' = 'APPROVED';
 export const TASK_NEW_COMPLETED: 'COMPLETED' = 'COMPLETED';
 export const TASK_NEW_INCOMPLETE: 'NOT_STARTED' = 'NOT_STARTED';
```

Comment:
Any chance we can converge on a consistent term/naming? `INCOMPLETE` <-> `NOT_STARTED`

Reply:
this is decided by the API

Reply:
* Oh I see, we can rename the variable to match the API value, yes